src/controllers/favoriteArticleController.js

```diff
@@ -0,0 +1,52 @@
+import FavoriteModelHelper from '../helpers/favorite';
+import ArticleModel from '../helpers/articles';
+
+/**
+ * @class FavoriteArticleController
+ */
+class FavoriteArticleController {
+  /**
+   *
+   * @param {*} request - request object
+   * @param {*} response - response object
+   * @returns {void} - redirects to article page
+   */
+  static async favoriteArticle(request, response) {
+    const { slug } = request.params;
+    const userId = request.userData.payload.id;
+    try {
+      const articleId = (await ArticleModel.queryForArticle(slug)).id;
+      await FavoriteModelHelper.favoriteArticle(articleId, userId);
+
+      response.redirect(`/api/articles/${slug}`);
+    } catch (error) {
+      response.status(500).json({
```

Comment:
HI Kizito, I like your implementations and I have learned a lot, but I would like you to stick with the team's error response from the helper's method [here](https://github.com/andela/balder-ah-backend/blob/staging/src/helpers/index.js).